src/color_grading_lut.shader

```diff
@@ -87,14 +94,14 @@ vec4 get_lut_mapping_trilinear(vec4 old_color){
 	ivec2 lut_color_cfc_pos = ivec2(int(lut_size*lut_color_cfc_vec.b + lut_color_cfc_vec.r), int(lut_color_cfc_vec.g));
 	ivec2 lut_color_ccf_pos = ivec2(int(lut_size*lut_color_ccf_vec.b + lut_color_ccf_vec.r), int(lut_color_ccf_vec.g));
 	ivec2 lut_color_ccc_pos = ivec2(int(lut_size*lut_color_ccc_vec.b + lut_color_ccc_vec.r), int(lut_color_ccc_vec.g));
-	vec3 lut_color_fff = texelFetch(lut, lut_color_fff_pos, 0).rgb;
-	vec3 lut_color_ffc = texelFetch(lut, lut_color_ffc_pos, 0).rgb;
-	vec3 lut_color_fcf = texelFetch(lut, lut_color_fcf_pos, 0).rgb;
-	vec3 lut_color_fcc = texelFetch(lut, lut_color_fcc_pos, 0).rgb;
-	vec3 lut_color_cff = texelFetch(lut, lut_color_cff_pos, 0).rgb;
-	vec3 lut_color_cfc = texelFetch(lut, lut_color_cfc_pos, 0).rgb;
-	vec3 lut_color_ccf = texelFetch(lut, lut_color_ccf_pos, 0).rgb;
-	vec3 lut_color_ccc = texelFetch(lut, lut_color_ccc_pos, 0).rgb;
+	vec3 lut_color_fff = convert_linear_to_srgb(texelFetch(lut, lut_color_fff_pos, 0).rgb);
+	vec3 lut_color_ffc = convert_linear_to_srgb(texelFetch(lut, lut_color_ffc_pos, 0).rgb);
+	vec3 lut_color_fcf = convert_linear_to_srgb(texelFetch(lut, lut_color_fcf_pos, 0).rgb);
+	vec3 lut_color_fcc = convert_linear_to_srgb(texelFetch(lut, lut_color_fcc_pos, 0).rgb);
+	vec3 lut_color_cff = convert_linear_to_srgb(texelFetch(lut, lut_color_cff_pos, 0).rgb);
+	vec3 lut_color_cfc = convert_linear_to_srgb(texelFetch(lut, lut_color_cfc_pos, 0).rgb);
+	vec3 lut_color_ccf = convert_linear_to_srgb(texelFetch(lut, lut_color_ccf_pos, 0).rgb);
+	vec3 lut_color_ccc = convert_linear_to_srgb(texelFetch(lut, lut_color_ccc_pos, 0).rgb);
```

Comment:
I decided to comment here instead of derailing https://gitlab.freedesktop.org/mesa/mesa/issues/1867

I think that instantly converting to sRGB here will lead to subtle error.
Later you are calling `get_interpolated_color` involving these values which are in sRGB color space however `get_interpolated_color` assumes linear color space. sRGB is not linear and cannot be interpolated that easily.